tests/comments/test_async_api.py

```diff
@@ -361,7 +361,7 @@ def test_fields(user, apiclient, question_ct, question):
     assert not response.data['use_org_terms_of_use']
 
     commentDict = response.data['results'][0]
-    assert len(commentDict) == 21
+    assert len(commentDict) == 22
     assert 'child_comments' in commentDict
     assert 'comment' in commentDict
     assert 'comment_categories' in commentDict
```

Comment:
I don't get why we only test a subset of the fields- Can we add all and also the new one?

Reply:
Yeah, to be honest I find that test a bit annoying, I had to update that number a hundred times already.. :sweat_smile: But I guess if we do test, it should be all fields, yes.

Reply:
Yeah, I started that, and then I thought, it doesn't make sense to test these fields here, but in the serializer test and then I started that and noticed, that we only do exclude the creator field instead of only adding the fields we actually want and then I just gave up and thought, that that's for another time. :woman_shrugging: